features/step_definitions/borrower_views_deed.rb

```diff
@@ -66,7 +65,7 @@
 
 Then(/^the borrower signature elements are present on page/) do
   @deed.borrowers.each do |borrower|
-    page.should have_content('[Awaiting digital signature of '\
+    page.should have_content('[Awaiting confirmation from '\
                              "#{borrower[:forename]} #{borrower[:surname]}]")
   end
 end
```

Comment:
Don't we have a replacement for this step that should be used instead of this to avoid duplication?